cumulus/xcm/xcm-emulator/src/lib.rs

```diff
@@ -360,12 +373,15 @@ macro_rules! decl_test_relay_chains {
 			#[derive(Clone)]
 			pub struct $name;
 
+			use $crate::NetworkComponent;
```

Comment:
This is causing `error[E0252]: the name `NetworkComponent` is defined multiple times`

Reply:
thanks!